runtime/src/main/java/org/corfudb/util/ImmutableListSetWrapper.java

```diff
@@ -14,13 +21,17 @@
  * An immutable set implementation that is backed by a list. This type is useful when
  * we have lists that we know conform to the set constraints, but creating the set is
  * too expensive.
- *
  */
+public class ImmutableListSetWrapper<E> extends AbstractSet<E> implements Set<E> {
+    private final List<E> internalList;
 
-public class ImmuableListSetWrapper<E> implements Set<E> {
-    final List<E> internalList;
+    public static <K, V> ImmutableListSetWrapper<Entry<K, V>> fromMap(@NonNull Map<K, V> original) {
+        return new ImmutableListSetWrapper<>(original.entrySet().stream().map(entry ->
```

Comment:
Please reformat code to something like
```
ImmutableList<SimpleImmutableEntry<Integer, Integer>> data = original
                .entrySet()
                .stream()
                .map(entry -> new SimpleImmutableEntry<>(entry.getKey(), entry.getValue()))
                .collect(ImmutableList.toImmutableList());

        return new ImmuableListSetWrapper<>(data);
```

current version is not quite well readable

Reply:
Good point, fixed.